src/Commands/sql/ContactFormSanitizeCommand.php

```diff
@@ -44,21 +55,39 @@ public function sanitize($result, CommandData $commandData) {
     /** @var \Drupal\contact\ContactFormInterface[] $contact_forms */
     $contact_forms = $this->entityTypeManager->getStorage('contact_form')->loadMultiple();
 
-    foreach ($contact_forms as $key => $contact_form) {
+    foreach ($contact_forms as $contact_form) {
       $is_corporate_form = $contact_form->getThirdPartySetting('oe_contact_forms', 'is_corporate_form', FALSE);
       if ($is_corporate_form) {
-        $messages = $this->entityTypeManager->getStorage('contact_message')->loadByProperties(['contact_form' => $contact_form->id()]);
-        foreach ($messages as $message) {
-          $message->set('name', 'User' . $message->id());
-          $message->set('mail', 'user+' . $message->id() . '@example.com');
-          $message->set('subject', 'subject-by-' . $message->id());
-          $message->set('message', 'message-by-' . $message->id());
-          $message->set('oe_country_residence', 'residence-in-' . $message->id());
-          $message->set('oe_telephone', '+000-' . $message->id());
-          $message->set('oe_topic', 'topic-' . $message->id());
-          $message->set('ip_address', '127.0.0.1');
-          $message->save();
-        }
+        $this->connection->update('contact_message')
```

Comment:
It works for the site's contact forms but personal contact forms are left unsanitized.
<img width="794" alt="Screenshot 2022-05-03 at 13 15 21" src="https://user-images.githubusercontent.com/1183951/166437287-ce7dbff4-57c7-413b-8f1d-3a6df4b4603b.png">